src/main/scala/io/github/davidgregory084/TpolecatPlugin.scala

```diff
@@ -129,7 +123,13 @@ object TpolecatPlugin extends AutoPlugin {
 
   override def projectSettings: Seq[Setting[_]] = Seq(
     Def.derive(
-      scalacOptions ++= scalacOptionsFor(scalaVersion.value, tpolecatScalacOptions.value)
+      scalacOptions := {
+        val previous   = scalacOptions.value
+        val scalaV     = scalaVersion.value
+        val filters    = scalacOptionsFor(scalaV, tpolecatExcludeOptions.value).toSet
+        val newOptions = scalacOptionsFor(scalaV, tpolecatScalacOptions.value)
+        (previous ++ newOptions).filterNot(filters).distinct
+      }
     ),
     tpolecatDevModeOptions := ScalacOptions.default,
```

Comment:
@DavidGregory084 I would also move this to build or global settings as "common" defaults. If one uses `ThisBuild / tpolecatDevModeOptions` it will have same problems as did `scalacOptions` in #60. Wdyt?

Reply:
Yeah we can definitely try that!